cli/hoodie-defaults.js

```diff
@@ -9,6 +9,7 @@ function getHoodieDefaults () {
     dbUrl: undefined,
     inMemory: false,
     loglevel: 'warn',
-    url: undefined
+    url: undefined,
+    applicationRoot: undefined
```

Comment:
> If the an app is included as a dependency, that the plugins should also be loaded as third party plugins, right?

I think It depends how we're introducing the app-come-module to hoodie, whether the consuming app will be using whatever API we eventually decide on for registering plugins with hoodie (which is what I think you're talking about here), or the consuming app doesn't know about hoodie at all and the app-come-module is loading hoodie itself. But we're probably not at the level of decoupling from Hapi to make the latter option a simple prospect.

> I can’t follow you hear. What do you mean with "if an application defined its own CLI application"?

In the same way that the now-deprecated hoodie-cli used to, if a consuming app has an executable globally installed via npm, it could potentially be run from any directory, making `process.cwd()` unreliable. We could forbid this usage for now though and consider reinstating this option if someone genuinely butts up against this use-case.